lib/classes/TradeOffer.js

```diff
@@ -263,7 +262,7 @@ function addItem(details, offer, list) {
 		assetid: (details.assetid || details.id).toString(), // always needs to be a string
 		appid: parseInt(details.appid, 10), // always needs to be an int
 		contextid: details.contextid.toString(), // always needs to be a string
-		amount: parseInt(details.amount || 1, 10), // always needs to be an int
+		amount: parseInt(details.amount || 1, 10) // always needs to be an int
```

Comment:
I ran some benchmarks:
```parseInt(int, 10): 0.000199 ms/iter
int << 0: 0.000003 ms/iter
parseInt(string, 10): 0.000085 ms/iter
string << 0: 0.000064 ms/iter```

In my opinion, it's really not worth it to make it less human-readable (or to define a redundant function for what already exists in the global namespace) for such a minute speed increase.